module/zfs/zio.c

```diff
@@ -1596,6 +1596,17 @@ zio_shrink(zio_t *zio, uint64_t size)
 	}
 }
 
+/*
+ * Get allocation size.
```

Comment:
This comment does not worth screen space it takes.  What would you say about: "Round provided allocation size up to a value that can be allocated by at least some vdev(s) in the pool with minimum or no additional padding and without extra space usage on others".